_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -0,0 +1,225 @@
+---
+id: oj04fd
+name: Element in sequential focus order has visible focus
+rule_type: atomic
+description: |
+  This rule checks that each element in sequential focus order has some visible focus indication.
+accessibility_requirements:
+  wcag20:2.4.7: # Focus Visible (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag22:2.4.11: # Focus Appearance (Minimum) (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag22:2.4.12: # Focus Appearance (Enhanced) (AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
```

Comment:
Note that this is currently not handled by the build. We'll need to grab WCAG 2.2 data, which is currently not done…

Reply:
Yeah, I'd leave this off until we can do the full migration to WCAG 2.2.